docs/common/craft-parts/reference/plugins/npm_plugin.rst

```diff
@@ -0,0 +1,59 @@
+.. _craft_parts_npm_plugin:
+
+NPM Plugin
```

Comment:
This is a great doc page, thanks a lot! Can you add a quick example at the end, showcasing some of the properties? 
some reference: https://github.com/canonical/craft-parts/blob/new-plugin-docs/docs/common/craft-parts/reference/plugins/go_plugin.rst?plain=1#L73